docs/pages.jl

```diff
@@ -0,0 +1,53 @@
+pages = [
+    "Home" => "index.md",
+    "Getting Started" => [
+        "Example 1: Linear Regression" => "getting_started/linear_regression.md",
+    ],
+    "Tutorials" => [
+        "Basics" => [
+            "tutorials/modeling_in_gen.md",
+            "tutorials/gfi.md",
+            "tutorials/combinators.md",
+            "tutorials/mcmc.md",
+            "tutorials/vi.md"
+        ],
+        "Advanced" => [
+            "tutorials/modeling_in_gen.md",
+            "tutorials/trace_translators.md",
+        ],
+        "Modeling Languages" => [
+        ],
+    ],
+    "How-to Guides" => [
+        "Custom Distributions" => "how_to/custom_distributions.md",
+        "Custom Modeling Languages" => "how_to/custom_dsl.md",
+        "Custom Gradients" => "how_to/custom_derivatives.md",
+        "Incremental Computation" => "how_to/custom_incremental_computation.md",
+    ],
+    "API Reference" => [
+        "Modeling Library" => [
+        "Generative Functions" => "api/model/gfi.md",
+        "Probability Distributions" => "api/model/distributions.md",
+        "Built-in Modeling Languages" => "api/model/modeling.md",
+        "Combinators" => "api/model/combinators.md",
+        "Choice Maps" => "api/model/choice_maps.md",
+        "Selections" => "api/model/selections.md",
+        "Optimizing Trainable Parameters" => "api/model/parameter_optimization.md",
+        "Trace Translators" => "api/model/trace_translators.md",
+        "Differential Programming" => "api/model/differential_programming.md"
```

Comment:
This should be "Differentiable programming"! In both the label and the filename.